<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParticlesDialog</class>
 <widget class="QDialog" name="ParticlesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Propriété des particules</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_37">
     <property name="text">
      <string>BulletSize</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_33">
     <item>
      <widget class="QDoubleSpinBox" name="bulletsize_x">
       <property name="minimum">
        <double>-99.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="bulletsize_y">
       <property name="minimum">
        <double>-99.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_26">
     <property name="text">
      <string>Texture</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_23" stretch="5,1">
     <item>
      <widget class="QLineEdit" name="texture_path"/>
     </item>
     <item>
      <widget class="QPushButton" name="texture_browse">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ressource.qrc">
         <normaloff>:/Medias/medias/explore.png</normaloff>:/Medias/medias/explore.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_25">
     <property name="text">
      <string>Number</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,1">
     <item>
      <widget class="QSpinBox" name="number">
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="build">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ressource.qrc">
         <normaloff>:/Medias/medias/cog.png</normaloff>:/Medias/medias/cog.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_20">
     <property name="text">
      <string>Gravity</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QDoubleSpinBox" name="gravity_x">
       <property name="minimum">
        <double>-99.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="gravity_y">
       <property name="minimum">
        <double>-99.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="gravity_z">
       <property name="minimum">
        <double>-99.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_30">
     <property name="text">
      <string>BoxSize</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_27">
     <item>
      <widget class="QDoubleSpinBox" name="boxsize_x">
       <property name="minimum">
        <double>-99.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="boxsize_y">
       <property name="minimum">
        <double>-99.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="boxsize_z">
       <property name="minimum">
        <double>-99.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_21">
     <property name="text">
      <string>FreeMove</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QDoubleSpinBox" name="freemove">
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_23">
     <property name="text">
      <string>LifeInit</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QDoubleSpinBox" name="lifeinit">
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_24">
     <property name="text">
      <string>LifeDown</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QDoubleSpinBox" name="lifedown">
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="QCheckBox" name="continuousmode">
     <property name="text">
      <string>Continuous  Mode</string>
     </property>
    </widget>
   </item>
   <item row="17" column="1">
    <widget class="QCheckBox" name="pointsprite">
     <property name="text">
      <string>Point Sprite Particules</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>label_23</zorder>
  <zorder>label_24</zorder>
  <zorder>label_26</zorder>
  <zorder>label_25</zorder>
  <zorder>label_20</zorder>
  <zorder>label_37</zorder>
  <zorder>label_30</zorder>
  <zorder>label_21</zorder>
  <zorder>freemove</zorder>
  <zorder>lifeinit</zorder>
  <zorder>lifedown</zorder>
  <zorder>continuousmode</zorder>
  <zorder>pointsprite</zorder>
 </widget>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
